File: app/src/main/java/com/ryanrvldo/moviecatalogue/data/source/remote/response/CreditsResponse.java
package com.ryanrvldo.moviecatalogue.data.source.remote.response;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;
import com.ryanrvldo.moviecatalogue.data.source.model.Cast;

import java.util.List;

import lombok.Data;

@Data
public class CreditsResponse {
    @SerializedName("id")
    @Expose
    private int id;

    @SerializedName("cast")
    @Expose
    private List<Cast> cast;
}

File: app/src/main/kotlin/com/ryanrvldo/moviecatalogue/data/remote/response/CreditsResponse.kt
package com.ryanrvldo.moviecatalogue.data.remote.response

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import com.ryanrvldo.moviecatalogue.data.model.Cast

data class CreditsResponse(
    @Expose @SerializedName("id") val id: Int,
    @Expose @SerializedName("cast") val cast: List<Cast>
)